src/ducks/sendPayment.ts

```diff
@@ -28,6 +29,7 @@ export const sendPaymentAction = createAsyncThunk<
         isPubnet: pubnet,
       });
     } catch (error) {
+      sentryCaptureException(error);
```

Comment:
this is the only spot I found where an error would not be caught by the logger

Reply:
I think it might be because we are logging the error on the UI side. 🤔 Can't think of any other reason.

Reply:
Actually, I just tested it and the error wasn't logged. @piyalbasu Could you please add it here? Then we can remove the Sentry from here. It was an oversight on my part for missing it here. 😬